cmd/server/main.go

```diff
@@ -1,53 +1,139 @@
 package main
 
 import (
+	"context"
 	"fmt"
-	"net"
-	"time"
+	"log"
+	"runtime"
 
+	"github.com/ethereum/go-ethereum/rpc"
 	"github.com/onflow/flow-evm-gateway/api"
+	"github.com/onflow/flow-evm-gateway/indexer"
+	"github.com/onflow/flow-evm-gateway/storage"
+	"github.com/onflow/flow-go-sdk/access/grpc"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/rs/zerolog"
+)
+
+const (
+	accessURL = "access-001.devnet49.nodes.onflow.org:9000"
 )
 
 func main() {
-	server := api.NewRPCServer()
-	defer server.Stop()
+	store := storage.NewStore()
+	runServer(store)
+	runIndexer(store)
+
+	runtime.Goexit()
+}
+
+func runIndexer(store *storage.Store) {
+	ctx := context.Background()
```

Comment:
I think we should pass the context to the method, so we can update it later and have cancel option

Reply:
Nice idea :+1:
Updated in https://github.com/onflow/flow-evm-gateway/pull/11/commits/42be5cdbb8ca4b9b948dbce13f7ab926f6298d9b